docs/pages/modules/module-api.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Module API
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
```

Comment:
we need to add a note here about this API being in alpha, people should expect it to change, and it may not have all of the features that you need yet. i'm happy to write this up later :)